### PR TITLE
Updated standard response format to either include :ok or :error symb…

### PR DESCRIPTION
**Pull request by enragedginger**
…ol. Removed System.halt buffoonery. Removed gitignore because Git uses the .gitignore.

### CODE REVIEW

File: gitignore [hunk 1]
```diff
@@ -1 +0,0 @@
-.DS_Store
```
Comment (himangshuj) [inline gitignore]: why did you change this

Comment (enragedginger) [reply gitignore]: It's a `gitignore` file which is useless. There *is* a `.gitignore` file though.

**Comment by enragedginger** [comment]
Closing because we decided not to use Atrium
